src/NJsonSchema.CodeGeneration.CSharp/Templates/Class.liquid

```diff
@@ -13,6 +13,9 @@
 {% if InheritsExceptionSchema -%}
 [Newtonsoft.Json.JsonObjectAttribute]
 {% endif -%}
+{% if IsDeprecated -%}
+[System.Obsolete{% if HasDeprecatedMessage %}("{{ DeprecatedMessage }}"){% endif %}]
```

Comment:
Here we need to escape the message - it might contains special chars which break the compiler. 
We can use this code here: 

![image](https://user-images.githubusercontent.com/2603405/85708467-7d58ed00-b6e4-11ea-9d08-0ea27b6d1392.png)

Simple way is to convert in the model class, the right solution would be to add a transformer like `| csharpdocs` and use it like this in the liquid tpl: 

![image](https://user-images.githubusercontent.com/2603405/85708672-b5f8c680-b6e4-11ea-98af-9a48e9efbda2.png)

![image](https://user-images.githubusercontent.com/2603405/85708810-de80c080-b6e4-11ea-9838-1c7972006ece.png)

e.g. `| literal`
e

Reply:
Added the transformer as I assume it might be useful in future.